Create add_content.md (#89)
elixir-europe/infectious-diseases-toolkit: .github/ISSUE_TEMPLATE/add_content.md

---
name: Add content request

> about: Suggestion of topic to be added to an existing page of the Infectious Diseases Toolkit

Reviewer: This does not sound like a correct English sentence or is it just me? :) Should it be Topic Suggestion for an existing page or topic to be added to an existing page or ?

> about: Suggestion of topic to be added to an existing page of the Infectious Diseases Toolkit

Author: ```suggestion
about: Topic suggestion for an existing page of the Infectious Diseases Toolkit
```

Does this work better?
